core/execution/engine.go

```diff
@@ -305,9 +399,22 @@ func (e *Engine) submitOrRestoreMarket(ctx context.Context, marketConfig *types.
 		e.log.Debug(msg, logging.Market(*marketConfig))
 	}
 
-	if err := e.submitMarket(ctx, marketConfig); err != nil {
+	parentMarketID := marketConfig.ParentMarketID
+	// propagate netparams unless this is a successor market
+	propagate := len(parentMarketID) == 0
+	if err := e.submitMarket(ctx, marketConfig, propagate); err != nil {
```

Comment:
Why do we not want to propagate the netparams for a successor market? Are we going to take the values from the parent or something?

Reply:
This is something that will be addressed in the next PR. I need to go through the netparams, and see which ones should be propagated, and which may not be ideal to propagate. Initially, in this PR, the successor market was a copy of the parent to which only the changes unique to the successor market were applied, and propagating network parameters would've been unnecessary because the parent market would've already held all the correct network parameters anyway.